src/platform/plugins/shared/discover/public/embeddable/components/search_embeddable_grid_component.tsx

```diff
@@ -81,7 +82,11 @@ export function SearchEmbeddableGridComponent({
     stateManager.rows,
     stateManager.totalHitCount,
     stateManager.columnsMeta,
-    stateManager.grid
+    stateManager.grid,
+    api.title$,
+    api.description$ ?? new BehaviorSubject(undefined),
```

Comment:
Are these truly optional?

Reply:
`SearchEmbeddableApi` extends `DefaultEmbeddableApi` which extends `DefaultPresentationPanelApi`. `DefaultPresentationPanelApi` defines  `Partial<PublishesTitle>`.  `SearchEmbeddableApi` could retype require `PublishesTitle` as none partial. Is that a change worth making? 

```
export interface DefaultPresentationPanelApi
  extends HasUniqueId,
    Partial<
      PublishesTitle &
        PublishesDataLoading &
        PublishesBlockingError &
        PublishesDescription &
        PublishesDisabledActionIds &
        HasParentApi<
          PresentationContainer & Partial<Pick<PublishesTitle, 'hideTitle$'> & PublishesViewMode>
        > &
        CanLockHoverActions
    > {}
```

Reply:
Resolved in f58f1575537a11d48848eb1472fde65dd29d7896